### PR TITLE
Post Model

### PR DESCRIPTION
**Pull request by SolomonMontagno**

### CODE REVIEW

File: server/src/models/Post.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const Model = require("./Model")
+
+
+class Post extends Model {
+    static get tableName() {
+        return "post"
```
Comment (DavidThomas-coder) [inline server/src/models/Post.js]: we want the table name to be plural, don't we?  "posts"

Comment (ToddLGarrison) [inline server/src/models/Post.js]: should be "posts"

**Review by ToddLGarrison — CHANGES_REQUESTED** [review]
Post -> Posts

**Review by ToddLGarrison — APPROVED** [review]
Looks good to me

**Pull request merged**